### PR TITLE
chore(mender-ci-tools): added to the release tool

### PR DESCRIPTION
**Pull request by 0lmi**
Changelog: None
Ticket: MEN-5959
Signed-off-by: Alex Miliukov <oleksandr.miliukov@northern.tech>

### CODE REVIEW

File: component-maps.yml [hunk 1]
```diff
@@ -722,3 +722,10 @@ docker_container:
     docker_image:
     - mender-gateway-qemu-commercial
     release_component: true
+
+  mender-ci-tools:
+    docker_image:
+    - mender-ci-tools
+    docker_container:
+    - mender-ci-tools
+    release_component: true
```
Comment (lluiscampos) [inline component-maps.yml:731]: Each components needs three entries: the git repo, the Docker image, and the container name. Check for example how it is done for `mtls-ambassador`

**Review by lluiscampos — CHANGES_REQUESTED** [review]

File: component-maps.yml (hunk 1 above)
Comment (0lmi) [reply component-maps.yml:745]: 👍

File: component-maps.yml [hunk 2]
```diff
@@ -276,6 +276,13 @@ git:
     release_component: false
     independent_component: true
 
+  mender-ci-tools:
+    docker_image:
+    - mender-ci-tools
+    docker_container:
+    - mender-ci-tools
+    release_component: true
```
Comment (lluiscampos) [inline component-maps.yml:284]:
```suggestion
  mender-ci-workflows:
    docker_image:
    - mender-ci-tools
    docker_container:
    - mender-ci-workflows
    release_component: true
```

File: component-maps.yml [hunk 3]
```diff
@@ -531,6 +538,13 @@ docker_image:
     - mender-reporting-indexer
     release_component: true
 
+  mender-ci-tools:
+    git:
+    - mender-ci-tools
+    docker_container:
+    - mender-ci-tools
+    release_component: true
```
Comment (lluiscampos) [inline component-maps.yml:546]:
```suggestion
  mender-ci-tools:
    git:
    - mender-ci-workflows
    docker_container:
    - mender-ci-workflows
    release_component: true
```

File: component-maps.yml [hunk 4]
```diff
@@ -722,3 +736,10 @@ docker_container:
     docker_image:
     - mender-gateway-qemu-commercial
     release_component: true
+
+  mender-ci-tools:
+    git:
+    - mender-ci-tools
+    docker_image:
+    - mender-ci-tools
+    release_component: true
```
Comment (lluiscampos) [inline component-maps.yml:745]:
```suggestion
  mender-ci-workflows:
    git:
    - mender-ci-workflows
    docker_image:
    - mender-ci-tools
    release_component: true
```

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Be careful with the name of the repo and the image/container. Something like this should do:

File: other-components-docker.yml [hunk 5]
```diff
@@ -4,3 +4,5 @@
 services:
     mtls-ambassador:
         image: registry.mender.io/mendersoftware/mtls-ambassador:mender-master
+    mender-ci-workflows:
+        image: registry.mender.io/mendersoftware/mender-ci-tools:master
```
Comment (lluiscampos) [inline other-components-docker.yml:8]: This image will go into Docker Hub, so it should be:
```suggestion
        image: mendersoftware/mender-ci-tools:master
```

On top of it, we'll use the `mender-1.2.3` kind of tags, so:
```suggestion
        image: mendersoftware/mender-ci-tools:mender-master
```

**Review by lluiscampos — CHANGES_REQUESTED** [review]
One more thing, sorry. We are missing is an update in `git-versions.yml`:

```
$ git diff
diff --git a/git-versions.yml b/git-versions.yml
index 073633e9..296982a5 100644
--- a/git-versions.yml
+++ b/git-versions.yml
@@ -60,3 +60,6 @@ services:
 
     mender-gateway:
         image: mendersoftware/mender-gateway:master
+
+    mender-ci-workflows:
+        image: mendersoftware/mender-ci-workflows:master
```

I figured these out running release tool commands on your PR. For example:
* release_tool --version-of mender-ci-tools
* release_tool --version-of mender-ci-workflows
* release_tool -m git mender-ci-workflows docker
* release_tool -m git mender-ci-workflows docker_url
* release_tool -m docker mender-ci-tools git

After this _I think_ we should be fine :sweat_smile:

**Review by lluiscampos — APPROVED** [review]

**Comment by 0lmi** [comment]
https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/682566805

**Comment by 0lmi** [comment]
https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/687530633

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**